Monika After Story/game/script-topics.rpy

```diff
@@ -2347,13 +2347,32 @@ label monika_rain_holdme:
     if mas_is_raining or mas_isMoniAff(higher=True):
         call monika_holdme_prep
         m 1eua "Of course, [mas_get_player_nickname()]."
+        $ start_hold_time = datetime.datetime.now()
         call monika_holdme_start
 
         call monika_holdme_reactions
 
         call monika_holdme_end
         # small affection increase so people don't farm affection with this one.
-        $ mas_gainAffection(modifier=0.25)
+        python:
+            # Monika likes being held for long periods of time, the longer the better
+            # We COULD use elapsed_time for this, but as it's a common var, we shouldn't
+            # in case it is being used by something else, so:
+            # TODO: zero-out, `del`, or `None`-ify elapsed_time when done with it in `monika_holdme_reactions`
+            delta_time = datetime.datetime.now() - start_hold_time
```

Comment:
That's the idea I had too. But it won't work because in `monika_holdme_reactions` you can spend unlimited time reading the dialogue. So this value isn't reliable either.